documentation/manual/src/ornate/installation.md

```diff
@@ -78,63 +73,78 @@ Add the following dependencies to your `client` sub-project:
 
 #### Client based on Play framework (JVM only)
 
-Add the following dependencies to your `client` sub-project:
+Add the following dependency to your `client` project:
 
 ~~~ scala expandVars=true
 libraryDependencies ++= Seq(
-  // client based on Play framework
-  "org.julienrf" %% "endpoints-play-client" % "{{version}}",
-  // (optional) JSON serialization using circe.io
-  "org.julienrf" %% "endpoints-play-client-circe" % "{{version}}"
+  "org.julienrf" %% "endpoints-play-client" % "{{version}}"
 )
 ~~~
 
-#### Client backed by scalaj-http
+#### Client backed by scalaj-http (JVM only)
+
+Add the following dependency to your `client` project:
 
 ~~~ scala expandVars=true
 libraryDependencies ++= Seq(
-  // client based on scalaj-http
-  "org.julienrf" %% "endpoints-scalaj-client" % "{{version}}",
-  // (optional) JSON serialization using circe.io
-  "org.julienrf" %% "endpoints-scalaj-client-circe" % "{{version}}"
+  "org.julienrf" %% "endpoints-scalaj-client" % "{{version}}"
 )
 ~~~
 
 #### Client backed by akka-http (JVM only)
 
+Add the following dependency to your `client` project:
+
 ~~~ scala expandVars=true
 libraryDependencies ++= Seq(
-  // client based on akka-http
-  "org.julienrf" %% "endpoints-akka-http-client" % "{{version}}",
-  // (optional) JSON serialization using circe.io
-  "org.julienrf" %% "endpoints-akka-http-client-circe" % "{{version}}"
+  "org.julienrf" %% "endpoints-akka-http-client" % "{{version}}"
+)
+~~~
+
+#### Client backed by sttp (JVM only)
```

Comment:
we need to upgrade, sttp can now be used for js as well (but not in our version, doc is correct ATM)